tests/PasswordGeneratorTest.php

```
@@ -0,0 +1,244 @@
+            20,
+            preg_match_all('/[!@#$%^&*()_\-=+;:,.?]/', $buffer),
+            'Should sample special characters repeatedly.',
+        );
+    }
+
+    public function testGenerateCharacterDistributionIncludesAllCharacterGroups(): void
+    {
+        $buffer = '';
+
+        for ($i = 0; $i < 10; $i++) {
+            $buffer .= PasswordGenerator::generate(50);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Helper\PasswordGenerator' in method 'testGenerateCharacterDistributionIncludesAllCharacterGroups'. (undefined)

(StaticAccess)